setup.py

```diff
@@ -57,6 +57,7 @@
     "invenio-i18n>=1.1.0,<1.2.0",
     "invenio-userprofiles>=1.0.1,<1.1.0",
     # https://github.com/inveniosoftware/invenio-indexer/commit/9749c2fe4e2cbaabc167ad7fb12ade945a2d580c
+    "oauthlib==2.1.0",
```

Comment:
Do not forget to remove this as we merged already a fix for that :)

Reply:
I have rebased 👍